src/components/auth/Login.js

```diff
@@ -0,0 +1,234 @@
+import { NavbarOther } from '../index';
+import PropTypes from "prop-types";
+import { login } from '../../state/actions/authAction';
+import { clearErrors } from '../../state/actions/errorAction';
+
+class Login extends Component {
+
+    state = {
+                email: '',
+                password: '',
+                message: null 
+            }
+    constructor(props){
+        super(props);
+         this.state = {
+           isDisabled:true,
+           email: '',
+           password: '',
+           message:null,
+         }
+         this.submitForm = this.submitForm.bind(this);                                              
+      }
+      validateEmail(email){
+        const pattern = /[a-zA-Z0-9]+[\.]?([a-zA-Z0-9]+)?[\@][a-z]{3,9}[\.][a-z]{2,5}/g;
+        const result = pattern.test(email);
+        if(result===true){
+          this.setState({
+            emailError:false,
+            email:email
+          })
+        } else{
+          this.setState({
+            emailError:true
+          })
+        }
+      }
+      validatePassword(password){
+         if(password.length>=5){
+           this.setState({
+             passwordError:false,
+             password:password
+           })
+         } else{
+           this.setState({
+             passwordError:true
+           })
+         }
+       }
+      static propTypes = {
+        isAuthenticated: PropTypes.bool,
+        error: PropTypes.object.isRequired,
+        login: PropTypes.func.isRequired,
+        clearErrors: PropTypes.func.isRequired
+      }
+
+      componentDidUpdate(prevProps) {
+        const { error, isAuthenticated } = this.props;
+        if(error !==prevProps.error) {
+          if(error.id === 'LOGIN_FAIL') {
+            this.setState({ message: error.message.error });
+          }else {
+            this.setState({ message: null });
+          }
+        }
+      }
+      handleChange(e){
+        const target = e.target;
+        const value = target.type === 'checkbox' ? target.checked : target.value;
+        const name = target.name;
+        this.setState({
+          [name]: value
+        });
+        this.setState({ [e.target.name]: e.target.value });
+        
+        if(e.target.name==='email'){
+         this.validateEmail(e.target.value);
+        }
+        if(e.target.name==='password'){
+          this.validatePassword(e.target.value);
+         }
+       if(this.state.emailError===false && this.state.passwordError===false){
+          this.setState({
+            isDisabled:false
+          })
+       }
+      }
+
+    // onChange = e => {
+    //            this.setState({ [e.target.name]: e.target.value });
+    //        };
+
+    submitForm(e){
+        e.preventDefault();
+        console.log("hello");
+       const { email, password } = this.state;
+
+       const user = {
+           email,
+           password
+       };
+       console.log(user.email);
+       console.log(user.password);
```

Comment:
Remove `console.log`s

Reply:
it was for testing purpose let's me remove them